### PR TITLE
CHE-2927: Add logs for active containers checked by DockerContainerCleaner.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What does this PR do?
Add logs for active containers checked by DockerContainerCleaner.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/2927

### Previous behavior
We logs clean up inactive containers. 

@garagatyi @mmorhun review please.

Signed-off-by: Aleksandr Andrienko <aandrienko@codenvy.com>

### CODE REVIEW

**Comment by AndrienkoAleksandr** [comment]
ci-build

File: plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/cleaner/DockerContainerCleaner.java [hunk 1]
```diff
@@ -80,6 +85,19 @@ public void run() {
...
         }
+        logActiveContainers(activeContainers);
+    }
+
+    private void logActiveContainers(List<String> activeContainers) {
+        if (activeContainers.isEmpty()) {
+            LOG.info("No active containers were detected.");
+        } else {
+            StringBuilder activeContainersReport = new StringBuilder();
+            activeContainersReport.append(activeContainers.size() > 1 ? "Containers are registered" : "Container is registered");
```
Comment (garagatyi) [inline plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/cleaner/DockerContainerCleaner.java]: please simplify it. It is just for maintenance purposes so we don't have to sync message with amount of containers

File: plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/cleaner/DockerContainerCleaner.java [hunk 2]
```diff
@@ -80,6 +85,19 @@ public void run() {
...
+            activeContainersReport.append(" in the api:");
+            activeContainers.forEach(containerName -> activeContainersReport.append("\n\t").append(containerName));
```
Comment (garagatyi) [inline plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/cleaner/DockerContainerCleaner.java]: Why not use list to string ?

**Review by garagatyi — CHANGES_REQUESTED** [review]

File: plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/cleaner/DockerContainerCleaner.java (hunk 1 above)
Comment (AndrienkoAleksandr) [reply plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/cleaner/DockerContainerCleaner.java]: Ok

**Review by garagatyi — APPROVED** [review]
Look good!

**Review by mmorhun — APPROVED** [review]

**Pull request merged**